ai_workloads/kueue/monitoring-pending-workloads.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="monitoring-pending-workloads-install-kueue"]
+= Monitoring pending workloads 
+:context: monitoring-pending-workloads
+
+toc::[]
+
+[role=”_abstract”]
```

Comment:
All of these need to be changed because you are using curly quotes and not straight which is throwing dita vale errors. Please update across the PR. You also need to be running ditavale checks on your PRs prior to submitting them.

Reply:
updated.